src/LandPlanner.cpp

```diff
@@ -100,28 +98,13 @@ bool LandPlanner::getTargetTwist(const ros::Time& time,
         return false;
     }
 
-    if(state_ == LandState::ACCELERATE_TO_DESCENT_VELOCITY) {
-        requested_z_vel_ = std::max(requested_z_vel_ - ((time - last_update_time_).toSec() * descend_acceleration_), descend_rate_);
-        if(requested_z_vel_ <= descend_rate_) {
-            state_ = LandState::DESCEND;
-        }
-    }
-    else if(state_ == LandState::DESCEND)
+    if(state_ == LandState::DESCEND)
```

Comment:
Why are we not ramping the descent rate anymore?

Reply:
We are still ramping the setpoint, its just that I didn't want to handle ramping 2 derivatives down. In theory this should be replaced by a plan generator in HLM. Which may happen. This gave acceptable performance in the sim.

Reply:
Done